docs/html/ux_research_design.rst

```diff
@@ -0,0 +1,67 @@
+====================
+UX Research & Design
+====================
+
+Over the course of 2020, the pip team has been working on improving pip's user
+experience.
+
+Currently, our focus is on:
+
+1. `Understanding who uses pip`_
+2. `Understanding how pip compares to other package managers, and how pip supports other Python packaging tools`_
+3. `Understanding how pip's functionality is used, and how it could be improved`_
+4. `Understanding how pip's documentation is used, and how it could be improved`_
+
+You can read the `overall plan`_ and the `mid-year update`_ to learn more about
+our work.
+
+How to contribute
+-----------------
```

Comment:
I think this and the *Next steps* section should be in the same level as *Other ways to contribute*.

Reply:
This looks fine to me.